pkg/daemon/daemon.go

```diff
@@ -964,6 +965,20 @@ func upgradeHackFor44AndBelow() error {
 	return nil
 }
 
+// Remove artifacts used by ignition, that the MCO should no longer
+// use since the machine is up.
+// Currently removes the systemd preset file written by Ignition.
+func removeIgnitionArtifacts() error {
+	_, err := os.Stat(constants.IgnitionSystemdPresetFile)
+	if err != nil && os.IsNotExist(err) {
+		return errors.Wrap(err, "failed to open Ignition-written systemd preset file")
```

Comment:
To be clear, your intent is to fail if the preset file doesn't exist?

In any event, the `os.Stat()` isn't necessary; you can just check for `os.IsNotExist()` from the `os.Remove()` call.

Reply:
No, I was going to do a rename so we keep a backup and then decided against it. It should not fail regardless. Fixed, thanks!